src/emphasis/alert/AlertItem.tsx

```diff
@@ -8,58 +8,50 @@
  *     http://www.apache.org/licenses/LICENSE-2.0
  */
 
-import * as React from 'react';
-import {ReactElement, ReactNode} from 'react';
+import * as React from "react";
+import {ReactElement, ReactNode} from "react";
 import {Icon} from "../../icon";
 import * as utils from "../../utils";
 import {AlertType} from ".";
 
 export type AlertItemProps = {
-    actions?: React.ReactElement
-    type?: AlertType
-    children?: React.ReactNode | React.ReactNode[]
-    icon?: React.ReactElement
-    static?: boolean
+    actions?: React.ReactElement;
+    type?: AlertType;
+    children?: React.ReactNode | React.ReactNode[];
+    icon?: React.ReactElement;
+    static?: boolean;
```

Comment:
Please add some documentation for this props

Reply:
Documentation done